simplify the homepage

afeld opened the pull request:
Inspired by https://github.com/18F/cg-landing/pull/37#issuecomment-168226981, this pull request encompasses a few things:
- Create a dedicated Help page, and expand on the different support channels – closes #153
- Remove the Help and Public Domain sections from the homepage, since those won't be applicable to first-time visitors
- Add a description of the different documentation sections on the homepage
- Add a CC0 badge to the footer

![screen shot 2015-12-31 at 3 45 07 pm](https://cloud.githubusercontent.com/assets/86842/12068155/7e8f0c56-afd5-11e5-85d2-dc0225da96e0.png)

/cc @jameshupp @brethauer

afeld committed:
add a dedicated help page - closes #153

afeld committed:
redo Help page to be driven by the type of question

afeld committed:
simplify title of Help page

afeld committed:
remove the 'send us feedback' link from the footer

afeld committed:
move the Public Domain dedication to the GitHub Repos page

afeld committed:
remove the Help section from the homepage

afeld committed:
add an overview of the sections on the homepage

afeld commented on content/help.md:
```diff
@@ -0,0 +1,25 @@
+
+## I'm interested in working on cloud.gov.
+
+Check out [18F's jobs site](https://pages.18f.gov/joining-18f/), and come talk to us in [our #devops-public Slack channel](https://chat.18f.gov/).
```
@mogul Does all of the information here seem accurate, for the current state of things? Any sorts of requests you can think of that I didn't mention? Is there too much?

dlapiduz replied:
I wonder if we should have 18F specific stuff in our docs. What do you think @mogul?

mogul replied:
I kind of agree, but let's make purging it a specific endeavor.

afeld commented on content/index.md:
```diff
@@ -6,14 +6,18 @@ type: index
+
+### Contributing
+
+Intended for cloud.gov operations staff, but made public for others to learn how we run the platform.
```
Don't love my various descriptions here...curious if others can think of better wording, or if they are helpful at all.

afeld commented on content/ops/repos.md:
```diff
@@ -27,3 +27,11 @@ Here are the repos with the tools and configuration for the platform:
+
+> This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+>
+> All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
```
This was a somewhat arbitrary choice of where to put this. Open to other ideas.

mogul committed:
Clearer statement of current status

mogul committed:
Minor elaboration

mogul committed:
Be more circumspect about what "cloud.gov" is

This (slightly ambiguous) wording leaves “cloud.gov” open to be the
“umbrella brand” in the future.
